lib/parseSty.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+
+var starts = ["\\newcommand",
+              "\\renewcommand",
+              "\\DeclareRobustCommand",
+              "\\defSpecFun"];
+
+function findBracketSections(string) {
```

Comment:
It would also be nice to have a test function for findBracketSections
In addition to prevent bugs, it also helps other to understand what the function does.